### ISSUE TITLE
Get illuminance value (LUX)

### ISSUE THREAD

**Issue by k4rm3l0** [zuvola/camera_worker#1]
Hello,

is there a way to get illuminance value (LUX format) of a picture from the stream?

My goal is to create a camera based light meter. 

Thank you

**Comment by zuvola** [comment]
This library only provides luminance values.
It appears that reflectance data is required to convert luminance to illuminance, but I am not familiar with the specifics.

**Comment by ibrahimbroachwala** [comment]
@k4rm3l0 Were you able to find a solution to build a camera based light meter ?

**Comment by k4rm3l0** [comment]
> @k4rm3l0 Were you able to find a solution to build a camera based light meter ?

@ibrahimbroachwala Yes, but [my code](https://github.com/k4rm3l0/camera_worker/commit/095a7b334237ac82f178a8876c840bf436066f59) is heavy dirty. It's just for learning purpose.

I found the formula to calculate the lux value [here](https://www.conservationphysics.org/lightmtr/luxmtr1.html).

**Comment by ibrahimbroachwala** [comment]
Thanks @k4rm3l0. I will check it out.
